### ISSUE TITLE
Who has not signed the CLA?

### ISSUE THREAD

**Issue by justinsb** [communitybridge/easycla#1674]
Is there any way to determine which Authors or Commits have not signed the CLA?

For example, I'm imagining an endpoint like this:  https://easycla.com/check/github.com/kubernetes-sigs/etcdadm/pull/168

That should return a list of emails from https://github.com/kubernetes-sigs/etcdadm/pull/168 that have not signed the CLA - or if that's considered problematic, just the commit SHAs.

I don't think this needs to be a private API, as anyone can push PRs to a k8s project and get the same information.

Does such an API exist?

**Comment by WillsonHG** [comment]
Hello @justinsb  We do have this implemented in the EasyCLA bot, see issue #308 It will list the commit SHA that is unauthorized. Unfortunately, there's nothing we can do for the older bot running on the Kubernetes repos, but there are plans to move Kubernetes to EasyCLA after some further testing.

For the v1 bot, I did a quick visual check through the commits and I noticed that included in the Jan 2019 commits were some from this user who doesn't have a github profile (if you mouse over their icon you can see it doesn't link to a profile). 
![image](https://user-images.githubusercontent.com/46732999/91600297-ba846a00-e935-11ea-9d74-13890aca4f55.png)

Basically, you want to be able to see a GitHub profile next to every commit made in the PR, otherwise, the bot is not going to be happy.

**Comment by dealako** [comment]
Won't fix. Out of scope of current product requirements/features.
